Dockerfile.art

```diff
@@ -22,16 +22,16 @@ FROM registry.redhat.io/ubi8/ubi-minimal:8.6-751
 
 COPY $REMOTE_SOURCES $REMOTE_SOURCES_DIR
 
-RUN microdnf install python39-pip python39-devel gcc-c++ && \
+RUN microdnf install -y python39-pip python39-devel gcc-c++ && \
     microdnf clean all && \
-    ln -sf /usr/bin/python3 /usr/local/bin/python
+    ln -sf /usr/bin/python3.9 /usr/local/bin/python
```

Comment:
Should we also check if /usr/bin/python3 exists and link it to python3.9 if not? Wondering if the need to update this link means the python3 isn't created (and whether that would cause a problem)

Reply:
/usr/bin/python3 should be there and by default it's point to python3.6, I don't think we need that extra login in dockerfile, we will find if if become failed to build

Reply:
Ok, thanks!